solc/CommandLineInterface.cpp

```diff
@@ -542,7 +545,8 @@ Allowed options)",
 			"Switch to linker mode, ignoring all options apart from --libraries "
 			"and modify binaries in place."
 		)
-		(g_argMetadataLiteral.c_str(), "Store referenced sources are literal data in the metadata output.");
+		(g_argMetadataLiteral.c_str(), "Store referenced sources are literal data in the metadata output.")
+		(g_argStandardJSON.c_str(), "Process standard JSON input / output.");
```

Comment:
I am a bit confused, but do not really see a point of having two implementations internally in the CLI, one using CompilerStack and the other using StandardCompiler.

I would just change it to use StandardCompiler only.

The above option though is not for switching between these implementation, but to accept a raw JSON input and output a raw JSON output.